hs-bindgen/fixtures/enums.dump.txt

```diff
@@ -1,20 +1,20 @@
-"ENUMS_H" :: "Invalid"
-"first" :: "Enum"
-  "FIRST1" :: "Int"
-  "FIRST2" :: "Int"
-"second" :: "Enum"
-  "SECOND_A" :: "Int"
-    _ :: "Int"
-      _ :: "Int"
-  "SECOND_B" :: "Int"
-  "SECOND_C" :: "Int"
-"same" :: "Enum"
-  "SAME_A" :: "Int"
-    _ :: "Int"
-  "SAME_B" :: "Int"
-    _ :: "Int"
-"packad" :: "Enum"
-  "packed" :: "Invalid"
-  "PACKED_A" :: "Int"
-  "PACKED_B" :: "Int"
-  "PACKED_C" :: "Int"
+"macro definition" "ENUMS_H" :: "Invalid"
+"EnumDecl" "first" :: "Enum"
+  "EnumConstantDecl" "FIRST1" :: "Int"
```

Comment:
does this mean that we don't need to check parent when getting enum value, checking the kind of the cursor is enough?

Reply:
Ah, yes, well-spotted. Will fix.